safe-core-sdk/api-kit/reference/migrating/v1.md

```diff
@@ -0,0 +1,114 @@
+# API Kit: Migrating from `safe-service-client`
```

Comment:
In the other migration guide we set `API Kit: Migrating from v1`. The `Migrating to v2` was set as the Link title in SUMMARY.md.

Should I align the titles or should I leave it like this?

Reply:
I think both the link title and the title were the same: https://docs.safe.global/safe-core-aa-sdk/api-kit/reference/v2.

Reply:
It seems gitdocs is ignoring the H1 element `#` in the file and overwritting with what you write in the SUMMARY file

Reply:
Ah, I see. Interesting. I am fine if you so it same as you did for the last one by changing it in the Summary file :)

Reply:
Yes, for the visuals in the menu `Migrating to v1` feels more correct, I'm just concerned because this migration also involved a package name change:

`safe-core-sdk` + `safe-ethers-lib` + `safe-web3-lib` -> `protocol-kit`
`safe-service-client` -> `safe-api-kit`

We can test this way and re-evaluate if people is having trouble finding it.

Reply:
Should this be added as a page description or somewhere in the text so that it appears when people search from the search bar? 👀

Reply:
Will think about it